apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -108,7 +106,7 @@ export class FilesStorageUC {
 	}
 
 	private async uploadFile(userId: EntityId, params: FileRecordParams, fileDescription: IFile) {
-		const [fileRecords] = await this.filesStorageService.getFilesOfParent(params);
+		const [fileRecords] = await this.filesStorageService.getFileRecordsOfParent(params);
 		const fileName = this.checkFilenameExists(fileDescription.name, fileRecords);
 		const entity = this.getNewFileRecord(fileName, fileDescription.size, fileDescription.mimeType, params, userId);
 		try {
```

Comment:
Wäre es konsequent diesen try catch in eine private funktion zu verschieben wie wir das im service gemacht haben?

Reply:
Ja genau, das dachte ich auch. Ist auch schon hier af8e0a089ca6a76fb465c8d6bd43fcb53ea337d7 passiert.